core-aam/core-aam.html

```diff
@@ -214,9 +214,24 @@
+      	<p>There are several concepts central to UI Automation:</p>
+	<ul>
+		<li>Automation element - controls and some application content is presented as automation elements.</li>
+		<li>Element properties - Automation elements have several common properties describing native framework element characteristics in agnostic way that all automation clients can understand. There are several ways to access element property values, described below.</li>
+		<li>Control Patterns - Some common interactivity in different frameworks is expressed as control patterns in UIA, allowing different automation client to interact with controls using common programmatic interfaces.</li>
+		<li>Events - Similarly to other accessibility APIs, automation elements support various events that allow automation providers notify clients on important state changes.</li>
+      	</ul>
+      	<p>All automation elements inherit from <code>IUIAutomationElement</code> interface and all properties that are not specific to particular control pattern can be queried through that interface. There are several ways to access UI Automation element properties:</p>
+	<ul>
+		<li>Direct property accessors to the current values - <code>Current{PropertyName}</code>, e.g. <code>IUIAutomationElement::CurrentName</code> for the <code>Name</code> property</li>
+		<li>Cached property accessors - <code>Cached{PropertyName}</code>, e.g. <code>IUIAutomationElement::CachedName</code> for the <code>Name</code> property. Using cached values is preferred when providers and clients are used in remote environments.</li>
+		<li><code>GetCurrentPropertyValue</code> and passing UIA Property ID enumeration value corresponding to that property to get current value, e.g. <code>IUIAutomationElement::GetCurrentPropertyValue(UIA_NamePropertyId)</code> for the <code>Name</code> property.</li>
+		<li><code>GetCachedPropertyValue</code> and passing UIA Property ID enumeration value corresponding to that property to get cached value, e.g. <code>IUIAutomationElement::GetCachedPropertyValue(UIA_NamePropertyId)</code> for the <code>Name</code> property.
+      	</ul>
+	<p>Properties for specific UIA control patterns are queried the same way using relevant control pattern interfaces. Taking Toggle Pattern as an example, to query ToggleState property clients can use IUIAutomationTogglePattern::CurrentToggleState or IUIAutomationTogglePattern::GetCurrentPropertyValue(UIA_ToggleToggleStatePropertyId) to get the current value.</p>
+      	<p>Note: The property mapping in this specification provide the <code>{PropertyName}</code> and don't specify all specific ways to access the property value. Automation clients can access current or cached values using conventions described above, depending on specific needs and coding style conventions.</p>
```

Comment:
I'm not sure, but I do not think W3C style allows contractions.  I will double check.  If not:
... and **do not** specify all specific ways ...